pyth_observer/calendar.py

```diff
@@ -0,0 +1,25 @@
+import datetime
+
+EQUITY_OPEN = datetime.time(9, 30, 0)
+EQUITY_CLOSE = datetime.time(16, 0, 0)
+EQUITY_EARLY_CLOSE = datetime.time(13, 0, 0)
+EQUITY_HOLIDAYS = ["17-01-2022", "08-02-2022", "21-02-2022", "15-04-2022", "30-05-2022",
+                   "20-06-2022", "04-07-2022", "05-09-2022", "24-11-2022", "25-11-2022", "26-12-2022"]
+EQUITY_EARLY_HOLIDAYS = ["08-02-2022", "25-11-2022"]
+
```

Comment:
You're gonna confuse the hell out of some Americans with this :)

I always liked YYYY-MM-DD since it is less ambiguous and is always an incrementing integer if you remove the dashes.

But regardless, this is a really weird way to do date comparisons, as strings. Don't you think? Why not make this a list of dates? You can still do > or < with datetimes?

Reply:
agree, will make it a datetime object

Reply:
why not work with some library to get you this holidays? this won't be true next week..
https://github.com/dr-prodigy/python-holidays

Reply:
ah that's because I got these holidays directly from NASDAQ as I wasn't sure that the NASDAQ holidays directly line up with US holidays -- can someone confirm this is the case?

Reply:
@erancx those are [NYSE holidays](https://www.nyse.com/markets/hours-calendars) aka when equity markets are closed, not national or state holidays.